.circle/buildenv_st2.sh

```diff
@@ -51,6 +58,13 @@ re="\\b$DISTRO\\b"
 
 ST2_CIRCLE_URL=${CIRCLE_BUILD_URL}
 
-write_env ST2_GITURL ST2_GITREV ST2PKG_VERSION ST2PKG_RELEASE DISTRO TESTING ST2_CIRCLE_URL
+# NOTE: We don't build Mistral package on Ubuntu Bionic
+if [ "${DISTRO}" = "bionic"} ]; then
+    ST2_PACKAGES="st2"
+else
+    ST2_PACKAGES=${ST2_PACKAGES:-st2 mistral}
```

Comment:
`buildenv_st2.sh` is intended to be st2-focused only, while `buildenv_mistral.sh` is about mistral only and so they're responsible for different packages but not alltogether.

This part I definitely don't like.

Will try different things to workaround it, - but in a meantime we have `buildenv_common.sh` which can be favored instead.

Reply:
Yeah, I'm also not a big fan of it either. For now, I will move it into ``buildenv_common.sh``.

Maybe eventually we can refactor how our docker-compose2.sh script works so it will only pass environment variable to docker compose if it's set.